tests/dir.props

```diff
@@ -67,6 +67,7 @@
 
    <ItemGroup> 
      <!-- Need to escape double forward slash (%2F) or MSBuild will normalize to one slash on Unix. -->
+     <DotnetSourceList Include="$(PackagesDir)AzureTransfer\$(__BuildType)\pkg\" Condition="'$(OverwriteCoreClrPackageVersion)' == 'true'" />
```

Comment:
Is there some way to share this directory? so we don't have it duplicated in multiple places?